kong/clustering/services/supported.lua

```diff
@@ -0,0 +1,8 @@
+-- DP and CP shares one list.
+-- Always from newer version to older version.
```

Comment:
```suggestion
-- Always order from most to least preferred version.
```
(we might consider some very new version "experimental" and put it after the "stable" one)

Reply:
From the beginning, I want to keep things simple. Probably the better way for "experimental" features is to put them into a separate experimental version.